x-pack/plugins/threat_intelligence/public/modules/indicators/components/flyout/indicator_value_actions/indicator_value_actions.tsx

```diff
@@ -63,6 +74,15 @@ export const IndicatorValueActions: VFC<IndicatorValueActions> = ({
   const copyToClipboardTestId = `${dataTestSubj}${COPY_TO_CLIPBOARD_BUTTON_TEST_ID}`;
   const popoverTestId = `${dataTestSubj}${POPOVER_BUTTON_TEST_ID}`;
 
+  if (readOnly) {
```

Comment:
I don't really like the prop name as it doesn't really reflect the reality. It's not like filter in/out are writing something somewhere. I have two options in mind:
1. the simplest is to rename the prop to just relfect the reality for now. Eg. `kqlBarIntegration: true/false`. This option is ok when we have 2 usecases, but if the component grow bigger it will become more confusing
2. make the component a bit more composable. Eg. pass an array of actions as a prop. The first two go as icons, the rest go into popover. If there are only two passed, then no "more actions" component is rendered. And then we can create two components `IndicatorValueActionsDefault` and `IndicatorValueActionsInCases` for example

Reply:
hey @maxcold, here's the update: I like the option #2 better and went that route, but after fighting it for a couple of hours I decided that it would be better use of my time to go with option #1. I was struggling to pass the component down via context and still be able to use the props. 
Option #1 does look perfect to me but it was quick to implement, and I feel like we can get to a better solution if one day the need comes.